src/main/scala/com/ebiznext/comet/job/index/bqload/BigQueryNativeJob.scala

```diff
@@ -127,6 +127,26 @@ class BigQueryNativeJob(
 }
 
 object BigQueryNativeJob extends StrictLogging {
+  def createTable(datasetName: String, tableName: String, schema: Schema): Unit = {
+    Try {
+      val bigquery = BigQueryOptions.getDefaultInstance.getService
+      val tableId = TableId.of(datasetName, tableName)
+      val table = scala.Option(bigquery.getTable(tableId))
+      table match {
+        case Some(tbl) if tbl.exists() =>
+        case _ =>
+          val tableDefinition = StandardTableDefinition.of(schema)
+          val tableInfo = TableInfo.newBuilder(tableId, tableDefinition).build
+          bigquery.create(tableInfo)
+          logger.info(s"Table $datasetName.$tableName created successfully")
+      }
+    } match {
+      case Success(_) =>
+      case Failure(e) =>
+        logger.info(s"Table $datasetName.$tableName was not created.")
+        Utils.logException(logger, e)
+    }
+  }
```

Comment:
Why did it become necessary to create the table after changing the dataset name, how was it created before?